test/unit/BPool/BPool_JoinPool.t.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
+import {BPoolBase} from './BPoolBase.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+import {BNum} from 'contracts/BNum.sol';
+import {IBPool} from 'interfaces/IBPool.sol';
+
+contract BPoolJoinPool is BPoolBase {
+  uint256 public poolAmountOut = 10e18;
+  // Assets Under Management
+  uint256 public tokenAUM = 10e18;
+  uint256 public secondTokenAUM = 30e18;
```

Comment:
nit: don't see how this is not just `balanceToken0` or sth like it

Reply:
_under management_ makes it clear that it's being managed by the pool imo(instead of calling it balance, where it could be anyone's balance, or more confusingly, a _bind_ balance passed to `bind`

Reply:
but it is indeed the token balance, perhaps `poolToken0Balance`? AUM seems like a new concept